0800-appendix-197559-brule.Rmd

```diff
@@ -49,16 +51,26 @@ provided in map attachment [`r fpr::fpr_my_bcfishpass(col_pull = dbm_mof_50k_gri
 fpr::fpr_table_bcfp(scroll = gitbook_on) 
 ```
 
+<br>
+
+## Stream Characteristics Upstream {.unnumbered}
+
+The stream was surveyed at two different sites upstream of crossing `r as.character(my_site)`. The first site was located approximately 1km from the culvert. This section was surveyed for `r fpr::fpr_my_priority_info(loc = 'us_ef1', col_pull = 'length_surveyed')`m (Figure \@ref(fig:photo-197559-01)). The second site was located approximately 1km upstream of modelled crossing 197536. This section was surveyed for `r fpr::fpr_my_priority_info(loc = 'us_ef2', col_pull = 'length_surveyed')`m to the location of a waterfall (Figure \@ref(fig:photo-197559-04)). This waterfall was approximately 5m in height, and blocks the upstream migration of all fish. This obstacle is listed in the Canadian Aquatic Barrier Database, as well as `bcfishpass` [@canadianwildlifefederation2023CanadianAquatic]. Stream habitat quality is shown in Figures \@ref(fig:photo-197559-02) - \@ref(fig:photo-197559-03).  
+
+<br>
+
 
 ## Fish Sampling {.unnumbered}
 
-Electrofishing was conducted at two sites upstream of the highway with no fish captured at either site Tables \@ref(tab:tab-fish-site-197559) - \@ref(tab:tab-fish-dens-197559). Habitat details are summarised in Table \@ref(tab:tab-habitat-summary-197559). 
+Electrofishing was conducted at two sites upstream of the highway with no fish captured at either site (Tables \@ref(tab:tab-fish-site-197559) - \@ref(tab:tab-fish-dens-197559)). Habitat details are summarised in Table \@ref(tab:tab-habitat-summary-197559). 
 
 `r if(gitbook_on){knitr::asis_output("<br>")} else knitr::asis_output("\\pagebreak")`
 
+<br>
+
 ```{r tab-habitat-summary-197559, eval=T}
 tab_hab_summary %>% 
-  filter(Site == my_site) %>% 
+  filter(Site %like% my_site) %>% 
```

Comment:
there is a good way to use stringr to get matched strings. We should start doing that and see if we can ditch data.table as a dependency. Used to need it to join tables that had geoms though...

https://stackoverflow.com/questions/22850026/filter-rows-which-contain-a-certain-string

`stringr::str_detect`

made an issue

https://stackoverflow.com/questions/22850026/filter-rows-which-contain-a-certain-string